docs/change-detection-strategy.md

```diff
@@ -0,0 +1,16 @@
+# Change Detection Strategy
```

Comment:
Can we add a section in the bottom regarding "Custom detections strategy", where we point out that it's recommended to stick to onPush, but possible to use `CheckAlways` for individual components.